src/sagemaker/clarify.py

```diff
@@ -326,6 +388,99 @@ class DatasetType(Enum):
     IMAGE = "application/x-image"
 
 
+class TimeSeriesJSONDatasetFormat(Enum):
+    """Possible dataset formats for JSON time series data files.
+
+    Below is an example ``COLUMNS`` dataset for time series explainability:
+
+    ```
+    {
+        "ids": [1, 2],
+        "timestamps": [3, 4],
+        "target_ts": [5, 6],
+        "rts1": [0.25, 0.5],
+        "rts2": [1.25, 1.5],
+        "scv1": [10, 20],
+        "scv2": [30, 40]
+    }
+
+    ```
+
+    For this example, JMESPaths are specified when creating ``TimeSeriesDataConfig`` as follows:
+
+    ```
+    item_id="ids"
+    timestamp="timestamps"
+    target_time_series="target_ts"
+    related_time_series=["rts1", "rts2"]
+    static_covariates=["scv1", "scv2"]
+    ```
+
+    Below is an example ``ITEM_RECORDS`` dataset for time series explainability:
```

Comment:
do we support item records or not? if we do, let's extend the schema accordingly, please

```
            SchemaOptional("dataset_format"): And(
                str,
                Use(str.lower),
                lambda s: s
                in (
                    "columns",
                    "timestamp_records",
                ),
            ),
```

Reply:
Good catch, will do (internally as well).